gc/base/MemoryPool.hpp

```diff
@@ -346,6 +346,49 @@ class MM_MemoryPool : public MM_BaseVirtual
 
 	MMINLINE virtual uintptr_t getDarkMatterSamples() { return _darkMatterSamples; }
 
+	MMINLINE virtual uintptr_t getFreeMemoryAndDarkMatterBytes() {
```

Comment:
this one does not have to be virtual (since it only exists here, as a simple helper to do a sum of 2 components), unlike the methods it relies on.

Reply:
MM_MemoryPoolBumpPointer has different version of getFreeMemoryAndDarkMatterBytes() right now, though MM_MemoryPoolBumpPointer is going to be removed in future.

Reply:
Hm, not sure what will happen while both coexist (before BumpPointer class is removed) - probably compiler will warn that BumpPointer one hides base one?. If so, leave it virtual - no much harm.